pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -328,6 +354,24 @@ func (plugin *kubenetNetworkPlugin) setup(namespace string, name string, id kube
 				return fmt.Errorf("Error setting promiscuous mode on %s: %v", BridgeName, err)
 			}
 		}
+
+		// configure the ebtables rules to eliminate duplicate packets by best effort
+		ebtablesPath, err := plugin.ebtables()
+		if err != nil {
+			glog.Warningf("Failed to get ebtables path: %v", err)
+		} else {
+			glog.Infof("Filtering packets with ebtables on mac address: %v, gateway: %v, pod CIDR: %v", macAddr.String(), plugin.gateway.String(), plugin.podCidr)
```

Comment:
I believe this should be safe for all modes, it should only be needed for promiscuous mode.